awx/ui_next/src/components/FormField/FormSubmitError.jsx

```diff
@@ -22,6 +22,8 @@ function FormSubmitError({ error }) {
       Object.values(error.response.data).forEach(value => {
         if (Array.isArray(value)) {
           messages = messages.concat(value);
+        } else {
```

Comment:
We have a util, which isn't in a util file, called getErrorMessage, which I think can handle all of these cases of error messages.  https://github.com/ansible/awx/blob/devel/awx/ui_next/src/components/ErrorDetail/getErrorMessage.js.  I thin we should, at some point, begin to migrate over to that file and let that handle our error messages.  If you want to investigate it, please feel free, but it isn't a blocker to merge this.

Reply:
Thanks for your suggestion. I looked in how we are handling this in other components. 

@keithjgrant filed https://github.com/ansible/awx/issues/7515, once this issue is sorted out, we will have a better pattern in how to deal those cases.